src/app/scam/components/shared/template-launch/template-launch.component.html

```diff
@@ -5,10 +5,7 @@ <h3>Launch template {{template?.Name}}</h3>
     <app-form-inventory *ngIf="template?.PromptInventory" [control]="form.Inventory"></app-form-inventory>
     <app-form-project-file *ngIf="template?.PromptPlaybook" [control]="form.Playbook" label="Playbook" [project]="form.get('Project').value"></app-form-project-file>
     <app-form-ssh-key *ngIf="template?.PromptSshKey" [control]="form.SshKey"></app-form-ssh-key>
-    <mat-form-field *ngIf="template?.PromptExtraVariables" class="block">
-      <mat-label>Extra variables</mat-label>
-      <textarea cdkTextareaAutosize matInput type="textarea" formControlName="ExtraVariables"></textarea>
-    </mat-form-field>
```

Comment:
I think that this should be kept, and merged with survey inputs when the user click "Launch"